datahub-project/datahub
# Nested domains cypress test

Test steps:
1. create a new domain without a parent
2. ensure the new domain has no parent in the navigation sidebar
3. move a domain from the root level to be under a parent
4. ensure it is no longer on the sidebar navigator at the top level but shows up under the parent
5. move a domain from under a parent domain to the root level
6. delete a domain
7. ensure the domain is no longer in the sidebar navigator

## Checklist
